### PR TITLE
Basic bot created

### PR DESCRIPTION
**Pull request by ghost**
Basic bot created with functions to detect mention spam and normal spamming of messages. On warnings, the bot will send a message to a moderators log for them to take further action.

### CODE REVIEW

File: events/ready.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const chainMessageAction = require('../eventActions/checkChainMessage');
```
Comment (Zaph-x) [inline events/ready.js]: Not really sure where this is coming from. I don't see the required file in any of the committed files. Maybe a leftover from Horace?

File: events/ready.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+module.exports = (client) => {
+	chainMessageAction.loadChainMessageTracker(client);
```
Comment (Zaph-x) [inline events/ready.js]: Same as above

File: eventActions/mentionSpam.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const config = require('../config.json')
+
+class mentionSpam {
+    static async preventMentionSpam(message) {
+        if (message.mentions.members.size >= 10) {
```
Comment (Zaph-x) [inline eventActions/mentionSpam.js]: Maybe the threshold should be a configurable variable instead of a seemingly constant variable?

**Review by Zaph-x — CHANGES_REQUESTED** [review]
Looking really solid. I've marked out three separate lines, where I think something could be changed.

One of them is a threshold, which I think could be a config variable 

The two others seem to be leftovers from Horace, that snuck into the code

File: events/ready.js (hunk 1 above)
Comment (ghost) [reply events/ready.js]: Yep, this is a leftover from copying over from Horace 😆 getting rid of it in next commit

File: eventActions/mentionSpam.js (hunk 3 above)
Comment (ghost) [reply eventActions/mentionSpam.js]: Good idea. I'll make this a variable in config.json.example.

File: events/ready.js (hunk 2 above)
Comment (ghost) [reply events/ready.js]: :+1:

**Comment by ghost** [comment]
Updated!

**Review by Zaph-x — APPROVED** [review]
Great work!

**Pull request merged**